pkgs/tools/archivers/undmg/default.nix

```diff
@@ -1,27 +1,27 @@
 { stdenv, fetchFromGitHub, zlib, bzip2 }:
 
 stdenv.mkDerivation rec {
-  version = "1.0.2";
+  version = "1.0.3";
   name = "undmg-${version}";
 
   src = fetchFromGitHub {
     owner = "matthewbauer";
     repo = "undmg";
-    rev = "refs/tags/v${version}";
-    sha256 = "0w9vwvj9zbpsjkg251bwv9y10wjyjmh54q2piklz74w64rlbqblr";
-    name = "undmg-${version}";
+    rev = "v${version}";
+    sha256 = "1pxqw92h2w75d4jwiihwnkhnsfk09cddh3flgrqwh9r3ry14fgbb";
+    
   };
 
   buildInputs = [ zlib bzip2 ];
 
-  setupHook = ./setup-hook.sh;
```

Comment:
Why did you remove the `setupHook`? It enables packages fetch `dmg` sources when this package is added to their buildInputs (eg. [musescore](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/musescore/darwin.nix#L14-L19)).